sway/handlers.c

```diff
@@ -840,6 +866,13 @@ static bool handle_pointer_button(wlc_handle view, uint32_t time, const struct w
 	// Check whether to change focus
 	swayc_t *pointer = pointer_state.view;
 	if (pointer) {
+		swayc_t *ws = swayc_parent_by_type(focused, C_WORKSPACE);
+		swayc_t *find = container_find(ws, &swayc_border_check, origin);
+		if (find != NULL) {
+			set_focused_container(find);
+			return EVENT_HANDLED;
+		}
+
```

Comment:
I don't think so.
If the user doesn't click on any title_bar process should continue.